From f64a8b85a45c238d01c80f113624109962fbbda9 Mon Sep 17 00:00:00 2001
From: Daniel Pavel <daniel.pavel@gmail.com>
Date: Mon, 22 Oct 2012 00:53:36 +0300
Subject: [PATCH] position the popup menu nicely

position the mounts popup menu using xfce_panel_plugin_position_menu,
instead of just the mouse click location
---
 panel-plugin/mount-plugin.c |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/panel-plugin/mount-plugin.c b/panel-plugin/mount-plugin.c
index 68f910b..7f636c0 100644
--- a/panel-plugin/mount-plugin.c
+++ b/panel-plugin/mount-plugin.c
@@ -351,8 +351,9 @@ on_button_press (GtkWidget *widget, GdkEventButton *event, t_mounter *mounter)
     {
 
         mounter_refresh (mounter); /* refreshs infos regarding mounts data */
-        gtk_menu_popup (GTK_MENU(mounter->menu), NULL, NULL, NULL, NULL, 0,
-                        event->time);
+        gtk_menu_popup (GTK_MENU(mounter->menu), NULL, NULL,
+                        xfce_panel_plugin_position_menu, mounter->plugin,
+                        0, event->time);
         return TRUE;
     }
     TRACE ("leaves on_button_press");
-- 
1.7.10.4

